src/sphinx/general/debugging.rst

```diff
@@ -0,0 +1,61 @@
+.. _debugging:
+
+#########
+Debugging
+#########
+
+Print Session Values
+====================
+
+If we wanted to print all values within a session we can just use this syntax:
+
+.. code-block:: scala
+
+    .exec { session => println(session); session }
+To print a single session value use syntax akin to this:
+  
+.. code-block:: scala
+
+    .exec({session => println(session("addComputer").as[String]); session }) 
```

Comment:
Actually, we have a trick there: Scala code samples are actually extracted from real classes that get compiled.
This way, we can make sure our samples are right, and up-to-date when we change something.
See `includecode` directive in [quickstart.rst](https://raw.githubusercontent.com/gatling/gatling/master/src/sphinx/quickstart.rst)  and how it works with [QuickStart.scala](https://github.com/gatling/gatling/blob/master/src/sphinx/code/QuickStart.scala).

Also, this code would be more simple the following way:

```scala
exec { session =>
  println(session("addComputer").as[String])
  session
}
```

I'd rather not encourage beginners to use `;` in Scala, it would bring more confusion.